packages/core/components/FileDetails/FileDetails.module.css

```diff
@@ -1,62 +1,22 @@
 .root {
-    --window-buttons-height: var(--url-bar-height); /* defined in App.module.css under .root{} */
-    --pagination-height: var(--url-bar-height); /* defined in App.module.css under .root{} */
+    --pagination-height: 40px;
     --padding: 10px;
     --resize-handle-width: 10px;
-    --max-detail-button-width: 250px;
-} 
 
-.expandable {
-    position: absolute;
-    right: 0;
-    height: calc(100% - var(--margin) * 2);
-    border: 1px solid var(--primary-background-color);
-    background-color: var(--secondary-background-color);
-    overflow: auto;
+    background-color: var(--primary-background-color);
+    color: var(--primary-text-color);
     display: flex;
     justify-content: center;
-    z-index: 1001;
-}
-
-.expandable::after {
-    background-image: linear-gradient(transparent, var(--secondary-background-color));
-    bottom: 0;
-    content: " ";
-    height: 50px;
-    right: 0;
-    max-width: 100%;
-    position: absolute;
-    width: 100%;
-    z-index: 11
+    min-width: 175px;
+    max-width: 95%;
+    overflow: auto;
+    width: var(--file-details-width); /* defined in App.module.css under .root{} */
```

Comment:
Getting some odd resizing behavior on this component where the contents resize but not the panel itself
<img width="222" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/5bcb53b7-0130-4d11-89c9-aa8dc3f39825">

Reply:
Are you shrinking the window or..?

Reply:
Window staying the same size, trying to increase the size of the file details one (dragging the left bar of the file details panel towards the center of the screen)

Reply:
Left this for round 2 because a lot of that panel needs to be refactored